xtask/src/commands/test.rs

```diff
@@ -34,6 +34,16 @@ pub struct Test {
     no_pre_compile: bool,
 }
 
+impl Default for Test {
+    fn default() -> Self {
+        Self {
+            no_demo: Default::default(),
+            with_demo: true,
```

Comment:
:thinking: .... so it's the default for `all` but not for `test`.

That doesn't sound okay.